uber/utils.py

```diff
@@ -1,28 +1,51 @@
 from uber.common import *
```

Comment:
Other than a bunch of formatting changes and the ``c`` refactor, nothing in this module has really changed.